models/issue.go

```diff
@@ -383,23 +383,18 @@ func newIssue(e *xorm.Session, repo *Repository, issue *Issue, labelIDs []int64,
 	}
 
 	// Check attachments.
-	attachments := make([]*Attachment, 0, len(uuids))
```

Comment:
I changed issues to match what I did for uploading files for repos, so that they had the same variables and settings as repos did.